igcollect/linux_pressure.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+"""igcollect - PSI - Pressure Stall Information
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+import os
+import time
+
+from argparse import ArgumentParser
+
+
+def parse_args():
+    parser = ArgumentParser(prog='linux_pressure.py')
+    parser.add_argument('--prefix', default='pressure')
+    return parser.parse_args()
+
+
+def main():
+    if os.path.isdir('/proc/pressure'):
+        args = parse_args()
+        for pressure in get_pressure_stats():
+            print('{}.{}'.format(args.prefix, pressure))
```

Comment:
Don't be afraid of early returns:

```suggestion
    if not os.path.isdir('/proc/pressure'):
        return
    
    args = parse_args()
    for pressure in get_pressure_stats():
        print('{}.{}'.format(args.prefix, pressure))
```

Reply:
Your not nested version is nicer! I agree. I adopted that code

Reply:
General idea is: Indent the exceptional/not interesting/not important code, and leave the normal/interesting/important code as near to the root indention as possible.